docs/modules/objectives/scoring.mdx

```diff
@@ -377,6 +377,43 @@ Result can be the name of a team, or one of the special values; "tie" for no win
         </td>
         <td>true</td>
       </tr>
+      <tr>
+        <td>
+          <label>overtime</label>
+        </td>
+        <td>Specify the length of the overtime once the tie is broken.</td>
+        <td>
+          <a href="/docs/reference/misc/time-periods">Time Period</a>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <label>max-overtime</label>
+        </td>
+        <td>Specify a maximum duration that the overtime should last.</td>
+        <td>
+          <a href="/docs/reference/misc/time-periods">Time Period</a>
+        </td>
+      </tr>
     </tbody>
   </table>
 </div>
+
+#### Overtime
+
+Overtime can be used in order to force a single winner in a match.  By using the optional `overtime="..."` attribute in the time element, you can specify a <a href="/docs/reference/misc/time-periods">time period</a>. If the score is tied once regulation time ends, the match will extend into overtime. The time will begin to tick down as soon as the tie is broken. It will reset back to the specified time if a tie is reached again before the time expires.
```

Comment:
Not only if a tie occurs again, if a "comeback" occurs it will also refresh. Imagine a TDM match with a scorebox that gives 2 points.
 - Team A: 10, Team B: 10
 - Overtime starts
 - Team A gets a kill: 11 to 10, overtime starts ticking down
 - Team B gets a score: 11 to 12, overtime refreshes and continues ticking down

Note that on that last score, overtime refreshed yet no tie existed

Reply:
@Pablete1234 Thanks for pointing that out.  I overlooked that.